extensions/wikia/PhalanxII/PhalanxStatsSpecialController.class.php

```diff
@@ -25,32 +27,61 @@ public function index() {
 		if ( strpos( $par, 'wiki' ) === 0 ) {
 			// show per-wiki stats
 			list ( , $wikiId ) = explode( "/", $par, 2 );
-			$this->blockWikia($wikiId);
+			$this->blockWikia( $wikiId );
 		} else {
-			$blockId = $this->wg->Request->getInt('blockId', intval($par));
+			$blockId = $this->wg->Request->getInt( 'blockId', intval( $par ) );
 			if ( !empty( $blockId ) ) {
 				// show block stats
-				$this->blockStats($blockId);
+				$this->blockStats( $blockId );
 			} else {
 				// show help page
-				$this->forward('PhalanxStatsSpecial', 'help');
+				$this->forward( 'PhalanxStatsSpecial', 'help' );
 			}
 		}
 
 		wfProfileOut( __METHOD__ );
 	}
 
-	private function blockStats($blockId) {
-		$this->wg->Out->setPageTitle( sprintf( "%s #%s", wfMsg('phalanx-stats-title'), $blockId ) );
+	public function help() {
+		if ( !$this->userCanExecute( $this->wg->User ) ) {
+			$this->displayRestrictionError();
+			return;
+		}
 
-		$data = Phalanx::newFromId($blockId);
+		$navLinksData = [
+			[
+				'text' => wfMessage( 'phalanx' )->escaped(),
+				'url' => $this->phalanxTitle->getFullURL(),
+			],
+		];
+		$this->wg->Out->prependHTML( $this->getNavLinks( $navLinksData ) );
+
+		$this->setVal( 'action', $this->title->getFullURL() );
+	}
+
+	private function blockStats( $blockId ) {
+		$this->wg->Out->setPageTitle( sprintf( "%s #%s", wfMsg( 'phalanx-stats-title' ), $blockId ) );
+
+		$navLinksData = [
+			[
+				'text' => wfMessage( 'phalanx' )->escaped(),
+				'url' => $this->phalanxTitle->getFullURL(),
+			],
+			[
+				'text' => wfMessage( 'phalanx-stats-title' )->escaped(),
+				'url' => $this->title->getFullURL(),
+			],
+		];
+		$this->wg->Out->prependHTML( $this->getNavLinks( $navLinksData ) );
```

Comment:
It's a bit frustrating I did not know about it, but I'll definitely give it a shot, thanks! :)